src/assets/sass/protocol/components/_breadcrumb.scss

```diff
@@ -0,0 +1,44 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import '../includes/lib';
+
+.mzp-c-breadcrumb {
+    background-color: get-theme('background-color-alt');
+    margin-bottom: 0;
+    padding: $spacing-sm $spacing-md;
+
+    .mzp-c-breadcrumb-item {
+        display: inline-block;
+        font-weight: 400;
+
+        &:last-child {
+            font-weight: bold;
+        }
+
+        + .mzp-c-breadcrumb-item {
+            &:before {
+                @include bidi(((content, "\2192", "\2190"),));
+                font-weight: normal;
+                margin: 0 .25em;
+            }
+        }
+
+        .mzp-c-breadcrumb-item-link {
+            text-decoration: none;
+        }
+    }
+
+    &.mzp-t-dark {
+        background-color: get-theme('background-color-alt');
```

Comment:
Not sure we want `-alt` here as it's pretty light: https://github.com/mozilla/protocol/blob/main/src/assets/sass/protocol/includes/_themes.scss#L56

<img width="776" alt="Screen Shot 2021-11-03 at 11 01 10 AM" src="https://user-images.githubusercontent.com/19650432/140049403-f2c4ca91-3e91-468c-b32c-7e9c9a8770fe.png">

Unless we also set color for `body-text-color-alt`? Or maybe change background to use `background-color-alt-inverse`?

Reply:
Oh, yep, just saw a comment on this in a previous review. Prefer `background-color-alt-inverse`